src/main/java/com/gooddata/notification/Subscription.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (C) 2004-2017, GoodData(R) Corporation. All rights reserved.
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE.txt file in the root directory of this source tree.
+ */
+package com.gooddata.notification;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonTypeInfo;
+import com.fasterxml.jackson.annotation.JsonTypeName;
+import org.springframework.web.util.UriTemplate;
+
+import java.util.List;
+
+/**
+ * Subscription for notifications
+ */
+@JsonTypeName("subscription")
+@JsonTypeInfo(include = JsonTypeInfo.As.WRAPPER_OBJECT, use = JsonTypeInfo.Id.NAME)
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class Subscription {
+
+    public static final String URI = "/gdc/projects/{project}/users/{user}/subscriptions";
+    public static final UriTemplate URI_TEMPLATE = new UriTemplate(URI);
+
+    private final List<Trigger> triggers;
+    private final Condition condition;
+    private final Template template;
+    private final List<String> channels;
+    private final Meta meta;
+
+
+    /**
+     * Creates Subscription
+     *
+     * @param triggers triggers of subscription
+     * @param condition condition under which this subscription activates
+     * @param template of message
+     * @param channels list of {@link ChannelConfiguration} uris
+     * @param meta information of subscription
+     */
+    @JsonCreator
+    public Subscription(@JsonProperty("triggers") final List<Trigger> triggers,
+                        @JsonProperty("condition") final Condition condition,
+                        @JsonProperty("message") final Template template,
+                        @JsonProperty("channels") final List<String> channels,
```

Comment:
such constructor should not be exposed to user. User should use ChannelConfiguration class

Reply:
done